src/applications/pre-need/utils/helpers.js

```diff
@@ -176,6 +176,27 @@ export const desiredCemeteryNoteDescriptionNonVeteran = (
   </va-alert>
 );
 
```

Comment:
Same issue as above.

"Unfortunately due to the nature of event driven programming this had to be 3 individual methods. Otherwise they all call at the same time and return the same value messing up the expected return value."